spec/abstract/AARCH64/VCPUAcc_A.thy

```diff
@@ -292,6 +259,42 @@ definition vcpu_switch :: "obj_ref option \<Rightarrow> (unit,'z::state_ext) s_m
               od))
      od"
 
+
```

Comment:
The extra whitespace here might have been unintended.

Reply:
I don't quite remember, but I think this one is actually intended for a change. It start a new section in the file.